java-ws/server/Todo/src/main/java/com/couchbase/lite/todo/model/TaskList.java

```diff
@@ -98,6 +100,8 @@ public static void delete(UserContext context, String id) throws CouchbaseLiteEx
             taskList.setName(r.getString(1));
             taskList.setOwner(r.getString(2));
             lists.add(taskList);
+
+            System.out.println("List query result to JSON (when get list): "+ r.toJSON());
```

Comment:
is this actually delete list?

Reply:
No this is in getTaskLists(), github just hides a big chunk of code above

Reply:
This is in method getTaskLists()

Reply:
kk!